iaac/terraform/aws-infra/rds/variables.tf

```diff
@@ -45,7 +45,7 @@ variable "db_allocated_storage" {
 variable "mysql_engine_version" {
   type        = string
   description = "The engine version of MySQL"
-  default     = "8.0.32"
+  default     = "8.0.33"
```

Comment:
updating here will not work because the parent stack is overriding the version, e.g. https://github.com/awslabs/kubeflow-manifests/blob/12b51ce2e010940a3cd25219567152330a89188d/iaac/terraform/aws-infra/rds/variables.tf#L48